packages/react-core/src/components/Wizard/WizardToggle.tsx

```diff
@@ -90,8 +94,14 @@ export const WizardToggle = ({
         aria-expanded={isNavExpanded}
       >
         <span className={css(styles.wizardToggleList)}>
-          <span className={css(styles.wizardToggleListItem)}>
-            <span className={css(styles.wizardToggleNum)}>{wizardToggleIndex}</span>{' '}
+          <span className={css(styles.wizardToggleListItem, status === 'error' && styles.modifiers.danger)}>
+            {status === 'error' ? (
+              <span className={css(styles.wizardNavLinkStatusIcon)}>
+                <ExclamationCircleIcon />
+              </span>
```

Comment:
Doesn't look like the toggle is reflecting the error status

<img width="517" alt="Screenshot 2024-05-17 at 7 02 26 PM" src="https://github.com/patternfly/patternfly-react/assets/35148959/93af4b41-eae0-4796-bacb-9b5c303585d4">